mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +291,63 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@requires_version('mffpy', '0.5.7')
+@requires_testing_data
+@pytest.mark.parametrize('idx, cond, tmax, signals, bads', [
+    (0, 'Category 1', 0.016, egi_txt_evoked_cat1_fname,
+     ['E8', 'E11', 'E17', 'E28', 'ECG']),
+    (1, 'Category 2', 0.0, egi_txt_evoked_cat2_fname,
+     ['E257', 'EMG'])
+])
+def test_io_egi_evokeds_mff(idx, cond, tmax, signals, bads):
+    """Test reading evoked MFF file."""
+    # Test reading all conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname)
+    assert len(evokeds) == 2
+    # Test reading list of conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname, condition=[0, 1])
+    assert len(evokeds) == 2
+    # Test invalid condition type
+    with pytest.raises(TypeError):
+        read_evokeds_mff(egi_mff_evoked_fname, condition=1.2)
+    # Test reading evoked data from single condition
+    evoked_cond = read_evokeds_mff(egi_mff_evoked_fname, condition=cond)
+    evoked_idx = read_evokeds_mff(egi_mff_evoked_fname, condition=idx)
+    for evoked in [evoked_cond, evoked_idx]:
+        assert evoked.comment == cond
+        assert evoked.nave == 3
+        assert evoked.tmin == 0.0
+        assert evoked.tmax == tmax
+    # Check signal data
+    data = np.loadtxt(signals, ndmin=2).transpose() * 1e-6  # convert to volts
+    assert_allclose(evoked_cond.data, data, atol=1e-6)
```

Comment:
For EEG data in volt units an `atol=1e-6` is a pretty bad tolerance level. Can this be `atol=1e-10` or `atol=1e-12` or something? It's going to depend a bit on the number of decimal places in the text file, but hopefully adjusting `atol` and/or `rtol=1e-4` (for example if the field width is 5 characters in the text file this would be a sensible choice) or something similar will help make it clearer the match is actually pretty good.